include/exadg/time_integration/time_int_gen_alpha_base.h

```diff
@@ -53,17 +53,26 @@ class TimeIntGenAlphaBase : public TimeIntBase
   set_current_time_step_size(double const & time_step_size) final;
 
   double
-  get_scaling_factor_mass() const;
+  get_scaling_factor_mass_from_acceleration() const;
+
+  double
+  get_scaling_factor_mass_from_velocity() const;
 
 protected:
   double
   get_mid_time() const;
 
   void
-  compute_const_vector(VectorType &       const_vector,
-                       VectorType const & displacement_n,
-                       VectorType const & velocity_n,
-                       VectorType const & acceleration_n) const;
+  compute_const_vector_acceleration_remainder(VectorType &       const_vector,
+                                              VectorType const & displacement_n,
+                                              VectorType const & velocity_n,
+                                              VectorType const & acceleration_n) const;
+
+  void
+  compute_const_vector_velocity_remainder(VectorType &       const_vector,
+                                          VectorType const & displacement_n,
+                                          VectorType const & velocity_n,
+                                          VectorType const & acceleration_n) const;
```

Comment:
Also here, I do not see the motivation to introduce a new function, which has the same interface as the pre-existing function. From a design perspective, it currently looks like combining the two functions into a single one would be more natural.